activerecord/lib/active_record/connection_adapters/postgresql/schema_definitions.rb

```diff
@@ -12,11 +12,21 @@ module ColumnMethods
         #   end
         #
         # By default, this will use the +gen_random_uuid()+ function from the
-        # +pgcrypto+ extension (only PostgreSQL >= 9.4), or +uuid_generate_v4()+
-        # function from the +uuid-ossp+ extension. To enable the appropriate
-        # extension, which is a requirement, you can use the +enable_extension+
-        # method in your migrations. To use a UUID primary key without any of
-        # of extensions, you can set the +:default+ option to +nil+:
+        # +pgcrypto+ extension (only for PostgreSQL >= 9.4). For PostgreSQL < 9.4,
```

Comment:
```
# +pgcrypto+ extension. As that extension is only available in
# PostgreSQL 9.4+, for earlier versions an explicit default can be set
# to use +uuid_generate_v4()+ from the +uuid-ossp+ extension instead:
```

Avoiding the double "PostgreSQL 9.4", and a "you" (though I see there's still one in the 'default: nil' bit. oh well.)